drones/main.py

```diff
@@ -4,22 +4,38 @@
 """
 
 import logging
-from connection import c  # Sample library import
+from drones.connection import connector
 
 
-def setup_log():
+def setup_log() -> None:
     """Setup global log config."""
 
-    logging.basicConfig(
-        filename="logfile.log",
-        filemode="w",
-        level=logging.DEBUG,
-        format="[%(filename)s] [%(levelname)s] : %(message)s",
-    )
+    fh = logging.FileHandler("logfile.log")
+    fh.setLevel(logging.DEBUG)
+    ch = logging.StreamHandler()
+    ch.setLevel(logging.DEBUG)
+    formatter = logging.Formatter("[%(filename)s] [%(levelname)s] : %(message)s")
+    fh.setFormatter(formatter)
+    ch.setFormatter(formatter)
+
+    logging.basicConfig(level=logging.DEBUG, handlers=[fh, ch])
     logging.info("Log initialized")
 
 
 if __name__ == "__main__":
     setup_log()
-    c.connect()  # Sample entrypoint call
-    # c.__track_errors()    # Sample invalid call - function inaccessible
+    log = logging.getLogger()
+
+    while True:
+
+        command_to_send = input("Input command: ")
+
+        if not command_to_send:

```

Comment:
`input()` will wait for user input, so the only way `command_to_send` can be empty is if he types an empty string. However, since you do not validate user's input anywhere, I don't think there is a reason to do that here. I think it should be possible to send an empty message to Tello to be consistent. Or at least warn user that because of empty command, it was not sent to Tello.

Reply:
Tello sends no response if you send an empty command so it is not sent. User warning is already included in send_command method so checking it in main loop is redundant. I will remove it.